File: java/src/main/java/vip/webmis/java/library/Inc.java
package vip.webmis.java.library;

import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;
import org.springframework.util.DigestUtils;

/* 公共函数 */
public class Inc {

  /* 格式化时间 */
  public static String date(String format){
    DateTimeFormatter df = DateTimeFormatter.ofPattern(format);
    LocalDateTime now = LocalDateTime.now();
    return df.format(now);
  }

  /* Md5加密 */
  public static String md5(String str){
    return DigestUtils.md5DigestAsHex(str.getBytes());
  }
  
}

File: python/app/library/Inc.py
import time
import hashlib

# 公共函数
class Inc:

  # 格式化时间
  def date(format='%Y-%m-%d %H:%M:%S',t=time.localtime()) :
    return time.strftime(format,t)

  # Md5加密
  def md5(str) :
    return hashlib.md5(str.encode(encoding='utf-8')).hexdigest()